### PR TITLE
Renaming repos

### PR DESCRIPTION
**Pull request by quarkytale**
#### Changes to Dockerfile, instances of repo references, READMEs and docs:
- `buoy_all.yaml` to `mbari_wec yaml `
- `buoy_entrypoint` to `mbari_wec`
- `buoy_sim` to `mbari_wec_gz`
- `buoy_msgs` in repo context to `mbari_wec_utils`

#### In discussion:
- `buoy_ws` to `mbari_wec_ws`
- Project name _Buoy Sim_ to ?

#### Workflow:
Rename repos on Github -> Merge in this PR -> Wait for green CI on https://github.com/osrf/buoy_sim/pull/139 -> Merge in https://github.com/osrf/buoy_sim/pull/139 -> Merge https://github.com/osrf/buoy_msgs/pull/45

### CODE REVIEW

**Review by mabelzhang — COMMENTED** [review]
I started reviewing the PR but realized that we never reached a final decision in the new names in #34. I posted on Slack to get a final decision.

I thought we wanted to cut out the `entrypoint` and just call it `mbari_wec`, but let's see what the group decision is. The `buoy_msgs` I thought we wanted to do `_interfaces` to align with ROS 2 naming convention.

I wonder if we should also change the workspace name from `buoy_ws` to `mbari_wec_ws`. That's debatable, since we're not changing any package names, just the repo names. That's another preference the group should decide on.

File: README.md
Comment (quarkytale) [reply README.md:56]: On line #46, Buoy Sim is used in the context of informal project name, could it be MBARI WEC Sim or MBARI Buoy Sim?

Comment (andermi) [reply README.md:56]: I think WEC Simulation rather than Sim

Comment (mabelzhang) [reply README.md:56]: MBARI WEC Gz Simulation, to be unambiguously clear? Honestly Sim means Simulation, so for somebody who doesn't know, they'll think WEC Sim and WEC Simulation are the same thing.
Basically we need to have an English, non-code name for the repo formerly called buoy_sim and now called mbari_wec_gz. The easiest thing to do is to call it what the code says, which is MBARI WEC Gazebo Simulation, with "Simulation" tacked on at the end for clarity.

Comment (quarkytale) [reply README.md:56]: There are a lot of instances addressing the project as Buoy Sim / Buoy Simulation, what's the verdict on that?

**Comment by quarkytale** [comment]
@mabelzhang @andermi Shall I go ahead with buoy_ws -> mbari_wec_ws, as it'll make the entrypoint, now mbari_wec and ws folder searchable together? That would mean Dockerfile updates as well

File: mbari_wec_all.yaml
Comment (quarkytale) [reply mbari_wec_all.yaml:3]: Yeah that's best, I'll update the yaml as well as build instructions

File: README.md
Comment (quarkytale) [reply README.md:126]: I can see those headings in here as well, you're talking about this right? 

> See [gz-math Python Get Started tutorial](https://github.com/gazebosim/gz-math/blob/gz-math7/tutorials/pythongetstarted.md). This step is needed until `PYTHONPATH` is automatically exported upstream, tracked in [this issue](https://github.com/osrf/mbari_wec_gz/issues/81)

File: mbari_wec_all.yaml
Comment (quarkytale) [reply mbari_wec_all.yaml:3]: Though one concern is there's nothing to build in that, what would `colcon` do in that case? Or there's some way by colcon ignore?

Comment (mabelzhang) [reply mbari_wec_all.yaml:7]: I don't think the COLCON_IGNORE is needed. I have that repo in my workspace, and since there's no CMake or package.xml, colcon doesn't do anything with it.

File: README.md
Comment (mabelzhang) [reply README.md]: Looks good now! Thanks!

**Review by mabelzhang — APPROVED** [review]
Thanks for the updates! README is looking good now.

The `COLCON_IGNORE` is not needed and can be removed - `colcon build` works for me without it. See https://github.com/osrf/mbari_wec/pull/38#discussion_r1154028966

I tested building the Dockerfile with the new workspace folder name (replaced `main` with temp branch name `renaming_repos` in this PR in the Dockerfile `wget` line).
With the new free surface hydrodynamics addition in https://github.com/osrf/mbari_wec/pull/37, the Dockerfile needed the new ppa lines, otherwise it couldn't build. We can replace with the alternative once https://github.com/osrf/mbari_wec_gz/pull/138 is in, but for now, we need these ppa lines for a working Dockerfile.
I added those in a commit 46819b5 to this PR.
Feel free to test that to cross check.

Don't forget to remove the COLCON_IGNORE before merging.

File: README.md
Comment (mabelzhang) [reply README.md]: Let's open an issue to decide that later and not let that block this PR.

Comment (mabelzhang) [reply README.md]: I updated the description of issue https://github.com/osrf/mbari_wec/issues/34 to reflect this.

File: mbari_wec_all.yaml
Comment (quarkytale) [reply mbari_wec_all.yaml:7]: Oh okay, removing it

**Pull request merged**
